Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -148,6 +148,93 @@ class CoreClient(BaseClient):
     def __init__(self, base_url: str, headers: dict, timeout: int = 120, proxy: bool = False, verify: bool = False):
         super().__init__(base_url=base_url, headers=headers, proxy=proxy, verify=verify)
         self.timeout = timeout
+    
+    def _http_request(self, method, url_suffix='', full_url=None, headers=None, json_data=None,
+                params=None, data=None, timeout=None, raise_on_status=False, ok_codes=None,
+                error_handler=None, with_metrics=False, resp_type='json'):
+        '''
+        """A wrapper for requests lib to send our requests and handle requests and responses better.
+
+            :type method: ``str``
+            :param method: The HTTP method, for example: GET, POST, and so on.
+
+            :type url_suffix: ``str``
+            :param url_suffix: The API endpoint.
+
+            :type full_url: ``str``
+            :param full_url:
+                Bypasses the use of self._base_url + url_suffix. This is useful if you need to
+                make a request to an address outside of the scope of the integration
+                API.
+
+            :type headers: ``dict``
+            :param headers: Headers to send in the request. If None, will use self._headers.
+            
+            :type params: ``dict``
+            :param params: URL parameters to specify the query.
+
+            :type data: ``dict``
+            :param data: The data to send in a 'POST' request.
+            
+            :type raise_on_status ``bool``
+                :param raise_on_status: Similar meaning to ``raise_on_redirect``:
+                    whether we should raise an exception, or return a response,
+                    if status falls in ``status_forcelist`` range and retries have
+                    been exhausted.
+            
+            :type timeout: ``float`` or ``tuple``
+            :param timeout:
+                The amount of time (in seconds) that a request will wait for a client to
+                establish a connection to a remote machine before a timeout occurs.
+                can be only float (Connection Timeout) or a tuple (Connection Timeout, Read Timeout).
+        '''
+        if not is_demisto_version_ge(version='8.6.0',build_number='924770'):
+            return BaseClient._http_request(self, method=method, url_suffix=url_suffix, full_url=full_url, headers=None,
+                                            json_data=json_data,params=params, data=data, timeout=timeout,
+                                            raise_on_status=raise_on_status)
+        try:
+            # Replace params if supplied
+            address = full_url if full_url else urljoin(self._base_url, url_suffix)
```

Comment:
does demisto._apiCall support full_url?

Reply:
we can also pass full url and also suffix to address

Reply:
we don't, thanks

Reply:
ok, was it tested?